telegram/chat.py

```diff
@@ -53,11 +53,11 @@ class Chat(TelegramObject):
     def __init__(self,
                  id,
                  type,
-                 title='',
-                 username='',
-                 first_name='',
-                 last_name='',
-                 all_members_are_admins=False,
+                 title=None,
+                 username=None,
+                 first_name=None,
+                 last_name=None,
+                 all_members_are_admins=None,
```

Comment:
We should follow their official API. 
We *can* do bkwd-compat for a while using **kwargs.
@jh0ker What do you think?